Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		046F007A2ED0030800F08512 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 046F00792ED0030800F08512 /* Images.xcassets */; };
```

Comment:
We should be able to revert this entire file with `git checkout main Demo/Demo.xcodeproj/project.pbxproj` - the add and commit that.